packages/io_library/lib/src/usecase/load_image_from_file_manager.dart

```diff
@@ -68,4 +70,16 @@ class LoadImageFromFileManager with LoggableMixin {
       }
     });
   }
+
+  Future<Image?> rebuildBackgroundImage(CatrobatImage catrobatImage) async {
+    Image? backgroundImage;
+    if (catrobatImage.backgroundImage.isNotEmpty) {
+      Uint8List? backgroundImageData =
+          base64Decode(catrobatImage.backgroundImage);
+      final result =
+          await imageService.import(Uint8List.fromList(backgroundImageData));
+      backgroundImage = result.unwrapOrElse((failure) => throw failure.message);
+    }
+    return backgroundImage;
+  }
```

Comment:
```suggestion
    if (catrobatImage.backgroundImage.isNotEmpty) {
      Uint8List? backgroundImageData =
          base64Decode(catrobatImage.backgroundImage);
      final result =
          await imageService.import(Uint8List.fromList(backgroundImageData));
     return result.unwrapOrElse((failure) => throw failure.message);
    }
    return null;
  }
```

Easier to read I think

Reply:
Also could " Uint8List? backgroundImageData" be "final backgroundImageData"?